openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -107,7 +107,7 @@ def get_tabs(self, course_overview):
         tabs = []
         for priority, tab in enumerate(get_course_tab_list(course_overview.effective_user, course_overview)):
             tabs.append({
-                'title': tab.title,
+                'title': tab.title or tab.get('name'),
```

Comment:
Is it okay that this falls back to `None` (should it be a blank string or some default value)?